src/resources/postcss/components/buttons/_solid.pcss

```diff
@@ -13,11 +13,12 @@
 		@mixin button-solid;
 
 		background-color: var(--color-accent-primary);
-		padding: 16px 22px 17px;
+		padding: 11px 22px 11px;
 		width: 100%;
 
 		@media (--viewport-medium) {
 			width: auto;
+			padding: 16px 22px 17px;
```

Comment:
check this padding again, i'm not sure this matches the design

Reply:
From what I checked it's still matching.

Reply:
So body-2 text line height is 23px, padding top is 16px, and padding bottom is 17px, total height I'm getting is 56px. In the designs I'm seeing a height of 45px.